src/components/Drawer.js

```diff
@@ -131,31 +126,31 @@ const Drawer = React.createClass({
           type='base'
         />
       </nav>
-    ) : <div style={styles.nav} />;
+    ) : <div className={css(styles.nav)} />;
   },
 
   render () {
     const styles = this.styles();
```

Comment:
Is it possible to run this in `componentWillMount` or `componentDidMount` and make it available on state?